controllers/storagecluster/kms_resources.go

```diff
@@ -35,10 +37,54 @@ var (
 type kmsConfigMapValidateFunc func(*corev1.ConfigMap) error
 
 var (
-	// just to make sure 'isKMSProviderReachable' function follows the validate function type
+	// just to make sure 'reachKMSProvider' function follows the validate function type
 	_ kmsConfigMapValidateFunc = reachKMSProvider
 )
 
+func deleteKMSResoucres(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) error {
```

Comment:
Can you pls add Uninstall keywords in the logs

Reply:
+1

Reply:
Done